Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocMultilineTextValue.css

```diff
@@ -7,36 +7,31 @@ span.bocMultilineTextValue
   position: relative;
   display: inline-block;
   vertical-align: middle;
-  overflow: hidden;
```

Comment:
can we do the min-height as 3*lineheight + 2*padding?

I'm not sure what x should be. it doesn't have to match the current height, instead it should be 3 lines + padding. Normally, I used 1.2em lineheight.

Reply:
I used _3 * lineheight + 3 * padding_ since the result looked better than only _2 * padding_. Other than that, I changed the lineheight to `1.2em`, as suggested.